Aspen-Discovery/aspen-discovery
# Where is it button renders consistently

Description:

Search results for which there are one or more copies available in one or more locations should all include a 'Where is it' button. With the current behaviour, the button only display for some of the search results which meet this criteria. This small patch resolves this by updating the check in copySummary.tpl to include one additional condition.

Test plan:

To replicate the bug:

- Run a search (keep the criteria field empty)
- Look through the search results: some results that have available copies at certain locations will display a 'Where is it?' button, whilst other won't

To test the fix:

- Apply the patch
- Run a search (keep the criteria field empty)
- Look through the search results: all results that have available copies at certain locations should now display a 'Where is it?' button
